src/QsCompiler/QirGeneration/Interop.cs

```diff
@@ -539,13 +539,8 @@ private IrFunction GenerateEntryPoint(string entryPointName, ArgumentTuple argum
 
                 // print the return value
                 var message = this.sharedState.GetOrCreateRuntimeFunction(RuntimeLibrary.Message);
-                var outputStr = QirExpressionKindTransformation.CreateStringLiteral(
-                    this.sharedState,
-                    result.QSharpType.Resolution.IsString ? "\"{0}\"" : "{0}",
-                    result);
+                var outputStr = QirExpressionKindTransformation.CreateStringLiteral(this.sharedState, "{0}", result);
```

Comment:
Before this change, if the type was string, it was wrapped around additional quotation marks. Why is it no longer the case?

Reply:
With the latest change, interpolation arguments of type string are wrapped inside quotation marks, hence there is no need to additionally wrap it once more.